idioms/return-consumed-arg-on-error.md

```diff
@@ -0,0 +1,63 @@
+# Return consumed argument on error
+
+## Description
+
+If a fallible function consumes an argument, return that argument back inside
+an error.
```

Comment:
Agree with the change to "consumed" instead of "moved".  But now I miss the Rust-specific nomenclature.  Can you tweak the description so it's not completely lost:

If a fallible function consumes **(moves)** an argument, return that argument back inside
an error.